src/frontend/js/components/SaleTunnel/index.full-process.spec.tsx

```diff
@@ -267,7 +267,36 @@ describe('SaleTunnel', () => {
     const $totalAmount = screen.getByTestId('sale-tunnel__total__amount');
     expect($totalAmount).toHaveTextContent(
       'Total' +
-        priceFormatter(product.price_currency, product.price).replace(/(\u202F|\u00a0)/g, ' '),
+        priceFormatter(product.price_currency, paymentPlan.price).replace(/(\u202F|\u00a0)/g, ' '),
```

Comment:
I know it's out of this PR scope, but WDYT about moving `.replace(/(\u202F|\u00a0)/g, ' ')`` to `priceFormatter` ?
This way we could use 
```suggestion
        priceFormatter(product.price_currency, paymentPlan.price),
```
everywhere in this file.